operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -248,20 +248,31 @@ local utils = import 'mixin-utils/utils.libsonnet';
       },
     },
 
-  resourcesPanelLegend(first_legend)::
+  resourceUtilizationAndLimitLegend(resourceName)::
     if $._config.deployment_type == 'kubernetes'
-    then [first_legend, 'limit', 'request']
+    then [resourceName, 'limit', 'request']
     // limit and request does not makes sense when running on baremetal
-    else [first_legend],
+    else [resourceName],
 
-  resourcesPanelQueries(metric, instanceName)::
+  resourceUtilizationQuery(metric, instanceName)::
```

Comment:
Why did you choose to not have requests/limits? I think we found them helpful when they were added on the other resource dashboards

I imagine because those will be different when this dashboard is used on a microservices deployment. In that case we can repeat the panel for every different container we have.

Alternatively, we can display the memory utilization as a % of the limit instead of an absolute value. I think I prefer this option.

Reply:
> I imagine because those will be different when this dashboard is used on a microservices deployment.

For this reason.

> In that case we can repeat the panel for every different container we have.

Well, that's goes against the idea of having an "Overview". If you want to go deep, you open the "Writes resources" or "Reads resources" dashboard.